public/manifest.json

```diff
@@ -1,25 +1,25 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "What's Near Me",
```

Comment:
I might be wrong but I think the short_name has a maximum recommended length of 12 characters?

Reply:
I see in MDN they recommend that the short_name not exceed 12 characters, but I am not finding the reasoning. No problem to shorten it, but I'm curious if anyone can shed light on the 12 character max? 
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/short_name

Below says it might be truncated if the situation requires it. Is this common?
https://developer.chrome.com/docs/extensions/mv2/manifest/name/

I will change it to "What's Near" for the short_name for now!

Reply:
My guess is that it will be truncated to fit the space

Reply:
Seems like it would be truncated in these situations:
https://developer.chrome.com/docs/extensions/mv2/manifest/name/#short_name